### ISSUE TITLE
Music doesnt pause during phone call

### ISSUE THREAD

**Issue by 3zozHashim** [auag0/DisableAudioFocus#3]
The module works perfectly great except for when i receive a phone call.

when i receive a phone call while listening to music, the music continues to play in the phone call unless i manually pause it.

**Comment by oenderg** [comment]
This is a very needed update. @auag0, is there a way to implement this?

**Comment by auag0** [comment]
i completely forgot, sorry for late, check it out!
maybe fixed
https://github.com/auag0/DisableAudioFocus/releases/tag/v1.1.0

**Comment by auag0** [comment]
I tried it myself, but it was completely pointless.
But please check it out

**Comment by oenderg** [comment]
> I tried it myself, but it was completely pointless. But please check it out

Yeah, unfortunately it doesn't seem to work, thanks for trying nonetheless. I think the issue lies in the fact that the original method "requestAudioFocus()" does more than just returning "AUDIO_FOCUS_REQUEST_GRANTED", like setting parameters or values that ensure the media playback is paused when a call is incoming. So, maybe it could be useful to look at the source code of that method, which I tried but I am not that savvy in Android programming haha

**Comment by merbah3266** [comment]
Will this problem be solved, or not?

**Comment by auag0** [comment]
> Will this problem be solved, or not?

i think no, i cannot do it, sorry
